jetty-websocket/websocket-jetty-tests/src/test/java/org/eclipse/jetty/websocket/tests/WebSocketOverHTTP2Test.java

```diff
@@ -246,8 +246,17 @@ public void testWebSocketConnectPortDoesNotExist() throws Exception
             wsClient.connect(wsEndPoint, uri).get(5, TimeUnit.SECONDS));
 
         Throwable cause = failure.getCause();
-        assertThat(cause, instanceOf(ConnectException.class));
-        assertThat(cause.getMessage(), containsStringIgnoringCase("Connection refused"));
+        cause.printStackTrace();
+        if (cause instanceof ClosedChannelException)
```

Comment:
This should be investigated. Connecting to a non-listening port should always report `ConnectException`.
Is it the JVM or our code creating the `ClosedChannelException`?

Reply:
Opened Issue #6660 to cover this case.
I'll revert these changes in this branch in the meantime.